### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by olisb**
my draft of the v1 Murmurations Map schema including all the fields we had in v0.1 but with only name and URL being required.

### CODE REVIEW

File: schemas/murmurations_map-v1.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "type": "object",
+  "properties": {
+    "linked_schemas": {
+      "$ref": "../fields/linked_schemas-v1.json"
+    },
+    "name": {
+      "$ref": "../fields/name-v1.json"
+    },
+    "url": {
+      "$ref": "../fields/urls-v1.json"
```
Comment (geoffturk) [inline schemas/murmurations_map-v1.json:15]: @olisb Do you want to collect only one URL or enable the user to enter multiple URLs? If the former, use `url-v1.json`. If the latter, rename the properties to `urls`.

**Review by geoffturk — CHANGES_REQUESTED** [review]
See above.

File: schemas/murmurations_map-v1.json (hunk 1 above)
Comment (olisb) [reply schemas/murmurations_map-v1.json:15]: URLs - done

**Review by geoffturk — APPROVED** [review]
LGTM 🚀

**Pull request merged**